openmapflow/ee_exporter.py

```diff
@@ -306,3 +317,33 @@ def export_for_labels(
                 ):
                     print(f"Started {exports_started} exports. Ending export")
                     return None
+
+
+class EarthEngineAPI:
```

Comment:
Is this class necessary for now? It seems like `get_ee_url` could just be a standalone function.

Reply:
It might make sense to move `ee.Initialize` to the top of this file so that any import of `ee_exporter` forces an initialization?